tests/integration/grpc_test.go

```diff
@@ -162,10 +164,11 @@ func templateAuthority(t *testing.T, pattern string, m *integration.Member) stri
 	return authority
 }
 
-func assertAuthority(t *testing.T, expectedAuthority string, clus *integration.Cluster, filterMethod string) {
+func assertAuthority(t *testing.T, expectedAuthorityPattern string, clus *integration.Cluster, filterMethod string) {
 	t.Helper()
 	requestsFound := 0
 	for _, m := range clus.Members {
+		expectedAuthority := templateAuthority(t, expectedAuthorityPattern, m)
```

Comment:
Since you introduce multiple calls in this patch, I think it makes sense to check `if requestsFound == 0` for each member. What do you think?

Reply:
agreed. Thanks for calling it out.